apprise/plugins/NotifyPushover.py

```diff
@@ -585,10 +590,16 @@ def parse_url(url):
                 NotifyPushover.unquote(results['qsd']['sound'])
 
         # Get the supplementary url
-        if 'url' in results['qsd'] and len(results['qsd']['url']):
-            results['url'] = NotifyPushover.unquote(results['qsd']['url'])
-        if 'url_title' in results['qsd'] and len(results['qsd']['url_title']):
-            results['url_title'] = results['qsd']['url_title']
+        if 'supplemental_url' in results['qsd'] and len(
+            results['qsd']['url']
+        ):
+            results['supplemental_url'] = NotifyPushover.unquote(
+                results['qsd']['url']
+            )
+        if 'supplemental_url_title' in results['qsd'] and len(
+            results['qsd']['url_title']
+        ):
+            results['supplemental_url_title'] = results['qsd']['url_title']
```

Comment:
Ah, so I think the corrected version should be as follows:

```suggestion
        if 'url' in results['qsd'] and len(
            results['qsd']['url']
        ):
            results['supplemental_url'] = NotifyPushover.unquote(
                results['qsd']['url']
            )
        if 'url_title' in results['qsd'] and len(
            results['qsd']['url_title']
        ):
            results['supplemental_url_title'] = results['qsd']['url_title']
```